ls1intum/Apollon
# Remove scale property from Apollon

**Disclaimer:**

This PR is based off of and includes changes from https://github.com/ls1intum/Apollon/pull/302. Reviewing this changes should become drastically easier once the base PR is merged.

### Checklist

### Motivation and Context



### Description
This PR removes the existing scale factor implementation entirely as it is no longer needed with the new canvas zooming approach.

### Steps for Testing
As this change does not change existing functionality nor add new functionality but rather only removes no longer used code, the most sensible way to test this change is by making sure the editor still behaves as expected.

### Test Coverage


| File | Branch | Line |
|------|-------:|-----:|

### Screenshots
